docs/book/intro.md

```diff
@@ -5,3 +5,132 @@ laminas-cli is a command-line interface for Laminas projects.
 It provides seamless integration with Laminas MVC and Mezzio applications.
 
 It is easily extensible by custom command. 
+
+## Integration with component
+
+If you'd like to add integration with laminas-cli into your components
+you have to the following:
+
+1. Use `laminas/laminas-cli` as dev dependency:
+
+```bash
+$ composer require --dev laminas/laminas-cli
+```
+
+2. Add the command to the library:
+
+```php
+namespace MyNamespace\Command;
+
+use Symfony\Component\Console\Command\Command;
+
+class MyCommand extends Command
+{
+    /** @var string */
+    protected static $defaultName = 'my-namespace:my-command';
```

Comment:
Why this, and not having a map of command name -> service pairs, such as the ContainerCommandLoader uses in symfony/console?

Reply:
Changed in the other PR per suggestion. No longer needed.